mindsdb/interfaces/agents/agents_controller.py

```diff
@@ -311,15 +313,37 @@ def get_completion(
         Raises:
             ValueError: Agent's model does not exist.
         '''
+        # Log the human question
+        human_question = "\n".join(f"{key}: {value}" for message in messages for key, value in message.items())
+        self.save_agent_history(
+            agent_id=agent.id,
+            type='question',
+            text=human_question,
+            sender_type='Human',
```

Comment:
are not `sender_type` and `type` duplicated? 
Human sends question, AI - completion